particle-accelerator/src/renderer/utilities/ParticleFunctions.js

```diff
@@ -0,0 +1,200 @@
+/*
+Library of functions which interact with the particle api to return/modify data.
+ */
+// eslint-disable-next-line import/no-import-module-exports
+import axios from 'axios'
+// eslint-disable-next-line import/no-import-module-exports
+import Product from './Product'
+
+const Particle = require('particle-api-js')
+
+const particle = new Particle()
+
+/**
+ * login: attempts an asynchronous login to a user's Particle account.
+ * @async
+ * @param username the user's username/email.
+ * @param password the user's respective password.
+ * @returns {Promise} a token in the event of a successful login or null in the
+ * event of an unsuccessful login (technically a promise)
+ */
+async function login(username, password) {
+  let token
+  try {
+    const loginData = await particle.login({ username, password })
+    token = loginData.body.access_token
+    return token
+  } catch (err) {
+    // console.error('Error in acquiring token: ', err)
```

Comment:
Why are these console errors commented out in this file?

Reply:
The linter wasn't a fan of them, and I know that console logs at least aren't considered good style, but I wasn't sure about console errors, so I commented them out and thought I'd ask you at a later date. What are your thoughts? Are they acceptable?